code/game/machinery/computer/camera.dm

```diff
@@ -46,6 +46,12 @@
 			M.unset_machine() //to properly reset the view of the users if the console is deleted.
 	return ..()
 
+/obj/machinery/computer/security/proc/isFarAway(var/atom/C)
+	var/turf/CONSOLETURF = get_turf(C)

```

Comment:
This is incredibly misleading if you don't actually look at where this proc is called. Please rename these variables and properly typecast the proc arguments to reflect the fact that this is comparing a security console to a camera.